joomla/joomla-cms
# Update layout login view for isis

## The problem

I discovered that the alignment on the login page of isis is not 100% pixel perfect. The field are just a few pixels to width. Hower, on RTL languages the wrong alignment was very good visible. So this patch solves alignment problems on LTR and RTL languages. I also add the possibility to load the `templates-rtl.css` in the login-view.
Farther optimizations:
- font-set icon `icon-joomla` instead of .png icon for the Joomla! logo on bottom of the loginform
- Full width login button
## Before/after screenshots

<b>LTR before</b>
![lrt-before](https://cloud.githubusercontent.com/assets/2659866/6654001/13369dc8-caa9-11e4-835b-f9f777f469d7.PNG)

<b>LTR after (very small difference on the right of the fields)</b>
![ltr after](https://cloud.githubusercontent.com/assets/2659866/6654003/2551c032-caa9-11e4-8aa5-c22b5b71dc2b.PNG)

<b>RTL before</b>
![rtl before](https://cloud.githubusercontent.com/assets/2659866/6654008/6f966e04-caa9-11e4-80b4-f5cf81baa7f2.PNG)

<b>RTL after</b>
![rtl after](https://cloud.githubusercontent.com/assets/2659866/6654020/44831144-caaa-11e4-8861-d9897690e35a.PNG)
## How to test this patch
1. Confirm the before/after screenshots on a LTR language (English)
2. Confirm 100% page width button and sharp Joomla! logo on bottom
3. Change your language to a RTL language (Persion for example)
4. Test step 2 and 3 again.
5. Confirm the login layout is also good shown on mobile devices. Feel free to use the Google Chrome F12 developer tools for this part
## Notes
- Also add some extra spaces for CS on some places
- As idea of @dgt41 the login button is full-width. Please let me know how you're thoughts are about that
